File: AgentApp/Backend/src/main/java/rent/app/model/Price.java
package rent.app.model;

import lombok.Data;

import javax.persistence.*;
import javax.validation.constraints.Max;
import javax.validation.constraints.Min;
import java.util.Set;

@Data
@Entity
@Table(name="Price")
public class Price {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column
    private String name;

    @Column
    private float pricePerDay;

    @Column
    private float insurancePrice;

    @Column
    private float distanceOverflowPrice;


    @Column
    private float discount;

    @Column
    private boolean isRemoved;



}

File: AgentApp/Frontend/rent-ng-app/src/app/model/price.ts
export class Price  {
  id: number;
  name: string;
  pricePerDay: number;
  insurancePrice: number;
  distanceOverflowPrice: number;
  discount: number;
  isRemoved: boolean;

  constructor(id: number, name: string, price: number, insurance: number, distance: number, discount: number, rem: boolean) {
    this.discount = discount;
    this.id = id;
    this.distanceOverflowPrice = distance;
    this.pricePerDay = price;
    this.insurancePrice = insurance;
    this.name = name;
    this.isRemoved = rem;
  }
}
